src/pages/en/guides/content-collections.md

```diff
@@ -241,21 +257,21 @@ const { post } = Astro.props;
 
 ## Rendering page content from entries
 
-To render the content of your Markdown and MDX entries, use the `<Content />` component returned by the `renderEntry()` function. This allows you to generate pages from your content entries (see [Generating pages from content collections](#generating-pages-from-content-collections)), add post previews to your homepage, or display your content elsewhere on your site.
+To render the content of your Markdown and MDX entries, use the `<Content />` component returned by the `render()` function. This allows you to generate pages from your content entries (see [Generating pages from content collections](#generating-pages-from-content-collections)), add post previews to your homepage, or display your content elsewhere on your site.
```

Comment:
Nice benefit of this API change: it’s easier to associate `render()` directly with a single entry in writing:

```suggestion
To render the content of a Markdown or MDX entry, use the `<Content />` component returned by its `render()` function. This allows you to generate pages from your content entries (see [Generating pages from content collections](#generating-pages-from-content-collections)), add post previews to your homepage, or display your content elsewhere on your site.
```